index.bs

```diff
@@ -2365,48 +2365,99 @@ Reviews and Review Responsibilities</h4>
-	including the general public,
-	have had adequate notice of the progress of the [=Working Group=]
-	(for example through notices posted to <a href="https://lists.w3.org/Archives/Public/public-review-announce/">public-review-announce@w3.org</a>)
-	and were able to actually perform reviews of and provide comments on the specification.
-	A second objective is to encourage groups to request reviews
-	early enough that comments and suggested changes
-	can still be reasonably incorporated in response to the review.
-	Before approving transitions,
-	the [=Director=] will consider who has been explicitly offered
-	a reasonable opportunity to review the document,
-	who has provided comments,
-	the record of requests to and responses from reviewers,
-	especially <a href="https://www.w3.org/Guide/process/charter.html#horizontal-review">W3C Horizontal Groups</a> [[CHARTER]]
-	and groups identified as dependencies in the charter
-	or identified as <a href="https://www.w3.org/2001/11/StdLiaison.html">liaisons</a> [[LIAISON]],
-	and seek evidence of clear communication to the general public
-	about appropriate times and which content to review
-	and whether such reviews actually occurred.
+	The objective of <dfn id=dfn-wide-review export>wide review</dfn> is to seek
+	feedback on a specification from the entire web community, including the general
+	public. The goal is to make the specification as robust as possible before it
+	becomes a [=W3C Recommendation=]. 
 
-	For example,
-	inviting review of new or significantly revised sections published in Working Drafts,
-	and tracking those comments
-	and the [=Working Group=]'s responses,
-	is generally a good practice which would often be considered positive evidence of wide review.
-	[=Working Groups=] <em class="rfc2119">should</em> announce to other W3C Working Groups
-	as well as the general public,
-	especially those affected by this specification,
-	a proposal to enter [=Candidate Recommendation=]
-	(for example in approximately 28 days).
-	By contrast a generic statement in a document
-	requesting review at any time
-	is likely not to be considered as sufficient evidence
-	that the group has solicited wide review.
-
-	A [=Working Group=] could present evidence that wide review has been received,
-	irrespective of solicitation.
-	But it is important to note that receiving many detailed reviews
-	is not necessarily the same as wide review,
-	since they might only represent comment
-	from a small segment of the relevant stakeholder community.
+	Wide review <em class="rfc2119">should</em> be sought continuously as a
+	specification matures, to allow different parts of the web community to iterate
+	on requirements, problems, and solutions. 
+
+	When a [=First Public Working Draft=] of a specification is published, a notification
+	<em class="rfc2119">must</em> be sent to <a href="https://lists.w3.org/Archives/Public/public-review-announce/">public-review-announce@w3.org</a>. 
```

Comment:
I don't think the Process needs to say who sends it or how, just that it must be sent. I'm also reticent about making anything dependent on tooling (especially tooling we may not have).

Reply:
I'm definitely not proposing that we enshrine any tool usage! I do think that where actions are required it is important to define in some way the scope of who has those actions. It could be as broad as the WG, I guess. 

The flip side of this is, this Process wording requires a thing to happen, but what would be the result if it did _not_ happen? Apparently nothing. We could add an evidence requirement that it was done as part of what is needed to request transition to CR, I guess.

Reply:
There's a little bit of technology dependency in relying on announcements to a particular mailing list, but that's relatively minor, easily fixable the day we want to move to something else than email, and pre-existing in the test, so let's leave that aside.

I do think it would be good to say whose responsibility it is to make that announcement, not to luck us into a particular mode of operation, but to be clear whose responsible. For example, if we say it's the Team that's in charge, then using an automated tool is fine, so is sending a mail manually, and so is noticing that the Editor already did it and leaving it there. But to avoid situations where everybody agrees it should be done, but all think that it's someone else's job, having someone be accountable is good.

I'd go with something like:
> When a [=First Public Working Draft=] of a specification is published
> the Team must ensure that a notification is sent to  <a href="https://lists.w3.org/Archives/Public/public-review-announce/">public-review-announce@w3.org</a>.

Reply:
I like that suggestion @frivoal .

Reply:
I've updated the text in both the wide and horizontal review sections to be:
>When the First Public Working Draft of a specification has been published, the Team must ensure that a notification is sent to >public-review-announce@w3.org.>

Hopefully this clarifies who has the responsibility, and also that the notification is expected once the FPWD *has been* published.